tcms/testruns/tests/test_admin.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# pylint: disable=invalid-name
+from django.urls import reverse
+from tcms.tests import LoggedInTestCase
+from tcms.utils.permissions import initiate_user_with_default_setups
+
+
+class TestTestRunAdmin(LoggedInTestCase):
+
+    @classmethod
+    def setUpTestData(cls):
+        super().setUpTestData()
+        initiate_user_with_default_setups(cls.tester)
+
+    def test_regular_user_can_not_add_view(self):
+        response = self.client.get(reverse('admin:testruns_testrun_add'))
+        self.assertRedirects(response, reverse('admin:testruns_testrun_changelist'))
+
+    def test_regular_user_can_not_change_views(self):
+        response = self.client.get(reverse('admin:testruns_testrun_change', args=[1]))
+        self.assertRedirects(response, reverse('testruns-get', args=[1]))
```

Comment:
TestRun with ID 1 doesn't obviously exist. Remove this entire method so we can drive this PR to completion. You will add it later - it needs more work.